t/typescript/typescript_3.9.9_ubi_8.3.sh

```diff
@@ -0,0 +1,64 @@
+#-----------------------------------------------------------------------------
+#
+# Package	:typescript
+# Version	: 3.9.9
+# Source repo	: https://github.com/Microsoft/TypeScript
+# Tested on	: ubi 8.3
+# Language      : node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Adilhusain Shaikh <Adilhusain.Shaikh@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME="TypeScript"
+PACKAGE_VERSION="v3.9.9"
```

Comment:
Make this configurable from command line.
```suggestion
PACKAGE_VERSION=${1:-v3.9.9}
```